demo/src/js/demo.js

```diff
@@ -1,8 +1,8 @@
-import numericycle from "DEGJS/numericycle";
-import { findAncestor } from "domUtils";
-import prism from "prism";
+import numericycle from "../../../src/numericycle.js";
+import { findAncestor } from "./domUtils.js";
+import Prism from 'prismjs';
```

Comment:
Nice addition to the demo. This will definitely help make the example code more readable!